etc/config/metadata/component-registries/elyra-kfp-url-preconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "display_name": "KFP Preloaded Components",
```

Comment:
Should this be 
```suggestion
  "display_name": "KFP Preloaded Components - URL",
```

As is there are now two entries with the same name:

![image](https://user-images.githubusercontent.com/13068832/135158490-b2296277-d330-4a4d-bcaa-022b58fae41b.png)


In addition `component-registries/elyra-kfp-preconfig.json` should probably be named `elyra-kfp-directory-preconfig.json` and use `KFP Preloaded Components - Directory` as display name.

Reply:
@kiersten-stokes

Reply:
I like that idea -- it slipped my mind sometime in the last few weeks that the Download Component will have to be permanent now

Reply:
I only brought this up because this is how it was done for Airflow and we should be consistent (:

Reply:
I completely agree! :)